### ISSUE TITLE
Dosent install with npm on Windows

### ISSUE THREAD

**Issue by cherifGsoul** [canjs/can-compile#19]
I just want to install it on Windows 7 and the command console shows errors :
```
> contextify@0.1.7 install C:\Users\b.cherif\AppData\Roaming\npm\node_modules\ca
n-compile\node_modules\jsdom\node_modules\contextify
> node-gyp rebuild


C:\Users\b.cherif\AppData\Roaming\npm\node_modules\can-compile\node_modules\jsdo
m\node_modules\contextify>node "C:\Program Files\nodejs\node_modules\npm\bin\nod
e-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild
gyp ERR! configure error
gyp ERR! stack Error: Can't find Python executable "python", you can set the PYT
HON env variable.
gyp ERR! stack     at failNoPython (C:\Program Files\nodejs\node_modules\npm\nod
e_modules\node-gyp\lib\configure.js:101:14)
gyp ERR! stack     at C:\Program Files\nodejs\node_modules\npm\node_modules\node
-gyp\lib\configure.js:64:11
gyp ERR! stack     at Object.oncomplete (fs.js:107:15)
gyp ERR! System Windows_NT 6.1.7601
gyp ERR! command "node" "C:\\Program Files\\nodejs\\node_modules\\npm\\node_modu
les\\node-gyp\\bin\\node-gyp.js" "rebuild"
gyp ERR! cwd C:\Users\b.cherif\AppData\Roaming\npm\node_modules\can-compile\node
_modules\jsdom\node_modules\contextify
gyp ERR! node -v v0.10.24
gyp ERR! node-gyp -v v0.12.1
gyp ERR! not ok
npm ERR! contextify@0.1.7 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the contextify@0.1.7 install script.
npm ERR! This is most likely a problem with the contextify package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls contextify
npm ERR! There is likely additional logging output above.

npm ERR! System Windows_NT 6.1.7601
npm ERR! command "C:\\Program Files\\nodejs\\\\node.exe" "C:\\Program Files\\nod
ejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "can-compile" "-g"
npm ERR! cwd E:\myapp
npm ERR! node -v v0.10.24
npm ERR! npm -v 1.3.21
npm ERR! code ELIFECYCLE
npm ERR!
npm ERR! Additional logging details can be found in:
npm ERR!     E:\myapp\npm-debug.log
npm ERR! not ok code 0
```

**Comment by daffl** [comment]
So it turned out that installing Python on Windows might solve the Contextify problem as well.

**Comment by pYr0x** [comment]
i have installed python 2.x and the problem is still unsolved....
there are a bunch of thread about the problem with contextify and windows. installing visual studio 2003 express might by a solution, but for me it is not (installing a programm with about 300 MB) für only one node packet

**Comment by daffl** [comment]
So... JSDOM 4.0 now works without contextify but only in io.js. Maybe a can-compile 1.0 is in order that uses JSDOM 4.0 and also works offline.

**Comment by pYr0x** [comment]
@daffl do you have any plans to remove contextify as a dependency for can-compile?
i installed oi.js but it seems that can-compile still uses contextify.
i want to compile my views like stache in a smaler production file

**Comment by pilou** [comment]
Any progress to pre-compile .mustache on windows 8.1 x64?
Workaround or something?

Should I use stache and require optimizer instead?

**Comment by daffl** [comment]
This should also be closed by using can-simple-dom via #45
